src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.bar--1 {
+  height: 70px;
+  display: flex;
+}
+
+.bar--2 {
+  height: 42px;
+  display: flex;
+}
+
+.bar {
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  box-shadow: 0 1 8px 0 10% #3d4e61;
+  border-radius: 4px;
+  width: 100%;
+}
+
+.input--big {
+  border: 0;
+  font-size: 16px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  text-align: left;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.input--small {
+  border: 0;
+  font-size: 14px;
+  font-weight: 400;
+  padding-top: 12px;
+  padding-bottom: 10px;
+  background-position: 13px 15px;
+  background-size: 11px;
+  text-align: left;
+}
+
+.input {
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  padding-right: 100vw;
+  padding-left: 62px;
+}
+
+.title {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.input:focus {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
```

Comment:
there is no shadow for the input in `focus` state, there is `linear-gradient` please check mock design